### PR TITLE
Added Hostel Inventory Page

### PR DESCRIPTION
**Pull request by chaithu10000**
* Removed unnecessary routes for reports and btech inventory pages

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -52,15 +49,6 @@ export default function App() {
             </Layout>
           }
         />
-
-        <Route
```
Comment (deveshidwivedi) [inline src/App.jsx:56]: Why did you delete the routes and inventory dashboard component?

Comment (chaithu10000) [reply src/App.jsx:56]: Because they are being rendered in the section navigation tab, please check

Comment (chaithu10000) [reply src/App.jsx:56]: Also there are some useless dependencies which creeped into the package.json without being used at a single place (check the blame to look yourself who did that)

**Comment by deveshidwivedi** [comment]
There are a lot of unnecessary changes in this PR, please add only the hostel inventory and if you feel there is need for other changes, open another PR. 
Also, resolve merge conflicts.

**Comment by chaithu10000** [comment]
Ok I will limit the scope of this PR and open another to resolve those problems 👍

**Comment by chaithu10000** [comment]
Had to force-push to just rewrite the remote commit history, made only the required changes, please review

**Comment by deveshidwivedi** [comment]
Few changes:

- You can remove the add product and request product pop ups, they are not supposed to look like this (see Figma). 
- We are currently only focusing on global admin pov, so we want "transfer product" not request product:
image

**Comment by chaithu10000** [comment]
Means should I completely remove them or when clicked they shouldn't show anything

**Comment by deveshidwivedi** [comment]
> Means should I completely remove them or when clicked they shouldn't show anything

Remove the pages that pop up, keeping buttons intact. Also change "Request Product" to "Transfer"

**Comment by chaithu10000** [comment]
In figma it is request item only so we can keep it the same or make it request product right

**Comment by chaithu10000** [comment]
I have made the necessary changes that you said once check it out

**Comment by deveshidwivedi** [comment]
> In figma it is request item only so we can keep it the same or make it request product right

I told you currently we are focused on global admin pov, he won't request items, he is responsible to transfer. Please make it "transfer".

**Comment by deveshidwivedi** [comment]
Also, please add margin at the bottom here and btech page, :
image

**Comment by chaithu10000** [comment]
I didn't understand what u said could you show where I should add the margin by marking it with some color
